### ISSUE TITLE
Decline in Accuracy scores

### ISSUE THREAD

**Issue by yuvrajhembade** [meituan/YOLOv6#389]
I am training the model for 400 epochs I got accuracy parameters after several eval intervals as below;

What could be the reason, I am not getting improvements in average precision & recall scores over epochs?

images for reference.

![image](https://user-images.githubusercontent.com/109716107/182328353-13de8ba3-4db2-497a-8bdb-8448c89c68e2.png)

![image](https://user-images.githubusercontent.com/109716107/182328658-0641bb38-a9fc-4650-952f-31649823ff01.png)

![image](https://user-images.githubusercontent.com/109716107/182328812-42cfbe52-2031-42df-8ebb-eba17096f17d.png)

Can anyone help?

**Comment by salkhokhar** [comment]
It's normal to not see any improvements after training beyond a certain point. Your model may have reached the point of [overfitting](https://www.geeksforgeeks.org/underfitting-and-overfitting-in-machine-learning/). 
Just use the weights with the best evaluation score for your inference. They will be located at /path-to-yolo-directory/runs/train/your-model-name/weights/best_ckpt.pt

**Comment by Zephyr69** [comment]
There are other factors which could lead to this, despite overfitting being the most common one. 

You need to look at val loss to confirm the overfitting, which yolov6 doesn't offer natively yet.

**Comment by alumini** [comment]
@yuvrajhembade, which config file are you using for training purpose?
